reprocessing.py

```diff
@@ -0,0 +1,51 @@
+from gladier.client import GladierClient
+# This is a HACK to enable glaider logging
+import gladier.tests
+from pprint import pprint
+
+
+class XPCSReprocessing(GladierClient):
+    client_id = 'e6c75d97-532a-4c88-b031-8584a319fa3e'
+    gladier_tools = [
+        'gladier_tools.manifest.tools.ManifestTransfer',
+        'gladier_tools.manifest.tools.ManifestToFuncXTasks',
+        'manifest_reprocessing.XPCSManifestTool',
+        'gladier_tools.xpcs.ApplyQmap',
+        'gladier_tools.xpcs.MakeCorrPlots',
+        'gladier_tools.xpcs.CustomPilot',
+    ]
+
+    flow_definition = 'manifest_reprocessing.XPCSManifestTool'
+    # flow_definition = 'gladier_tools.xpcs.ApplyQmap'
+    # flow_definition = 'gladier_tools.xpcs.EigenCorr'
+    # flow_definition = 'gladier_tools.xpcs.MakeCorrPlots'
+    # Warning: Renames dataset to "mydataset_qmap"!
+    # flow_definition = 'gladier_tools.xpcs.CustomPilot'
```

Comment:
Each of these flow_definitions should be runnable, given the inputs to the flow are valid. I still don't have a 'full' flow that encompasses all of these.